lib/Secrets/extensionplugins.h

```diff
@@ -219,6 +237,12 @@ class SAILFISH_SECRETS_API AuthenticationPlugin : public QObject
     virtual Sailfish::Secrets::AuthenticationPlugin::AuthenticationTypes authenticationTypes() const = 0;
     virtual Sailfish::Secrets::InteractionParameters::InputTypes inputTypes() const = 0;
 
+    virtual bool supportsLocking() const;
+    virtual bool isLocked();
```

Comment:
I would suggest `const` for all `isLocked()` methods, I see there are a few of them, but I didn't comment on every one.

Reply:
Yep, done